rllib/algorithms/appo/tf/appo_tf_rl_module.py

```diff
@@ -0,0 +1,51 @@
+from typing import List
+
+
+from ray.rllib.algorithms.ppo.tf.ppo_tf_rl_module import PPOTfRLModule
+from ray.rllib.core.models.base import ACTOR
+from ray.rllib.core.models.tf.encoder import ENCODER_OUT
+from ray.rllib.core.rl_module.rl_module_with_target_networks import (
+    RLModuleWithTargetNetworks,
+)
+from ray.rllib.policy.sample_batch import SampleBatch
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.framework import try_import_tf
+from ray.rllib.utils.nested_dict import NestedDict
+
+_, tf, _ = try_import_tf()
+
+OLD_ACTION_DIST_KEY = "old_action_dist"
+OLD_ACTION_DIST_LOGITS_KEY = "old_action_dist_logits"
+
+
+class APPOTfRLModule(PPOTfRLModule, RLModuleWithTargetNetworks):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        catalog = self.config.get_catalog()
+        self.old_pi = catalog.build_pi_head(framework=self.framework)
```

Comment:
Can you comment these? I'm not sure what this "old" stuff is for :D

Reply:
Actually, I see, this is just the old Policy stuff. Never mind, then, about adding the comments (will be removed very soon anyways).

Reply:
done.